### PR TITLE
Navebar + Main-section

### PR DESCRIPTION
**Pull request by ali-techworld**
**Create a mobile-friendly website using**

1. HTML
2. CSS

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="nav">
+        <div class="logo">Ali-tech</div>
+        <img src="/img/Union.png" alt="nave-icon" class="handburger">
+    </div>
```
Comment (hiromataba) [inline index.html:14]: - [ ] For your toolbar The container of the section should be a header tag. Typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="nav">
+        <div class="logo">Ali-tech</div>
+        <img src="/img/Union.png" alt="nave-icon" class="handburger">
```
Comment (hiromataba) [inline index.html:13]: - [ ] The logo should be on the left and has a link (a tag).
- This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.

Comment (hiromataba) [inline index.html:13]: - [ ] The hamburger button is on the right, and should have a nav tag.

- The nav tag is for semantics reason, and the right position for convention and user experience for mobile devices.

- It is not showing for now unfortunately , please fix it your image is broken 

image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    <a href="#" class="connect">LET’S CONNECT</a>     
+   
+    <ul class="icon">
+    <li class="twiter"><a href="#"><img src="/img/twiter.svg" alt=""></a></li>
+    <li class="linkdin"><a href="#"><img src="/img/Linkedin icon.svg" alt=""></a></li>
+    <li class="m"><a href="#"><img src="/img/m.svg" alt=""></a></li>
+    <li class="github"><a href="#"><img src="/img/github.svg" alt=""></a></li>
+    <li class="v"><a href="#"><img src="/img/v.svg" alt=""></a></li>
+</ul> 
+</body>
```
Comment (hiromataba) [inline index.html:29]: - [ ] Please make sure to have your design look like the original , using same margin, padding,background,images,.....

- Original Design 👇 


image


- Your design 👇 


image

**Review by hiromataba — CHANGES_REQUESTED** [review]
Hi @ali-techworld   :wave: :wave: 

almost-there

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Your PR has a proper title and description  :+1: 
- Your linters are passing :heavy_check_mark:  
- Profesional README file :rocket: 

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by shahadat3669 — CHANGES_REQUESTED** [review]
Hi @ali-techworld , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.



## Invalid Code Review Request

You have submitted a project, that is invalid because of linter errors.
- To ensure that your code adheres to the project's linting standards, it is important to have a green linter result. This means that the last commit in your pull request should have a green check mark (:white\_check\_mark:) or a summary indicating that all the required checks have passed. This helps in maintaining a clean and consistent codebase.

| Your | Professional |
| --- | --- |
| image | image |

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**